content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -0,0 +1,370 @@
+---
+title: Memory Manager
+
+reviewers:
+- klueska
+- derekwaynecarr
+- TBD
+
+content_type: task
+min-kubernetes-server-version: v1.21
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.21" >}}
+
+The *Memory Manager* enables the feature of guaranteed memory (and hugepages) allocation for pods in the Guaranteed [QoS class](/docs/tasks/configure-pod-container/quality-service-pod/#qos-classes). You can select from two memory allocation strategies. The first one, the *single-NUMA* strategy, is intended for high-performance and performance-sensitive applications. The second one, the *multi-NUMA* strategy, complements the overall design while it overcomes the situation that cannot be managed with the *single-NUMA* strategy.

```

Comment:
We do not really have two strategies, in general, we have only two policies, none and static, all other things defined on the topology manager level(like restricted vs single NUMA node). We have the default behavior in the case when the topology manager runs with the none policy and the behavior similar to one provided by the topology manager restricted policy.